diff --git a/node_modules/react-native-web/dist/exports/TextInput/index.js b/node_modules/react-native-web/dist/exports/TextInput/index.js
index 1234567..abcdefg 100644
--- a/node_modules/react-native-web/dist/exports/TextInput/index.js
+++ b/node_modules/react-native-web/dist/exports/TextInput/index.js
@@ -270,7 +270,6 @@ var TextInput = /*#__PURE__*/React.forwardRef((props, forwardedRef) => {
   function handleKeyDown(e) {
     var hostNode = e.target;
     // Prevent key events bubbling (see #612)
-    e.stopPropagation();
     var blurOnSubmitDefault = !multiline;
     var shouldBlurOnSubmit = blurOnSubmit == null ? blurOnSubmitDefault : blurOnSubmit;
     var nativeEvent = e.nativeEvent;
